clang/lib/Basic/Targets/PPC.h

```diff
@@ -364,7 +364,14 @@ class LLVM_LIBRARY_VISIBILITY PPCTargetInfo : public TargetInfo {
   // have Glibc since it is Glibc that provides the HWCAP[2] in the auxv.
   static constexpr int MINIMUM_AIX_OS_MAJOR = 7;
   static constexpr int MINIMUM_AIX_OS_MINOR = 2;
-  bool supportsCpuSupports() const override { return getTriple().isOSGlibc(); }
+  bool supportsCpuSupports() const override {
```

Comment:
> Does this need to be a separate function if it is the same as `supportsCpuIs()`? Can they be combined?

yes, we can combined them , but it need another NFC to do it. since not only PPC.h has the two function but also X86.h

Reply:
This sounds reasonable to me, thanks!